Marlin/src/module/probe.cpp

```diff
@@ -512,7 +512,7 @@ static bool do_probe_move(const float z, const float fr_mm_m) {
  * @param  short_move Flag for a shorter probe move towards the bed
  * @return The raw Z position where the probe was triggered
  */
-static float run_z_probe(const bool short_move=true) {
```

Comment:
The description of the short_move parameter above needs to be removed

Reply:
It seems like there were calls to `run_z_probe` that passed a parameter. Did all of those get cleaned up?

Reply:
Only one function calls `run_z_probe`, and that's `probe_pt`.

Reply:
Ahh. Ok. I was thinking of `probe_pt`. Sorry about that.